src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_Layout.cshtml

```diff
@@ -44,39 +34,15 @@
             </div>
         </nav>
     </header>
-    <div class="container">
-        <partial name="_CookieConsentPartial" />
-        <main role="main" class="pb-3">
-            @RenderBody()
-        </main>
```

Comment:
Does this mean the page body won't render anymore?

Reply:
Ugh yea you are right, thanks for the quick catch. I fixed this in the 3 layout views